WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -508,6 +510,17 @@ import WordPressComAnalytics
     }
 
 
+    // Refresh the header of a site topic when returning in case the
+    // topic's following status changed.
+    func refreshTableHeaderIfNeeded() {
+        if let siteTopic = readerTopic as? ReaderSiteTopic {
+            if let header = tableView.tableHeaderView as? ReaderStreamHeader {
```

Comment:
Will do.  I also think a `guard` would be more appropriate now that I'm giving it a second look, so I'll also make that change.